### ISSUE TITLE
Default init_printing() fails in jupyter

### ISSUE THREAD

**Issue by chakpongchung** [sympy/sympy#9646]
```
init_printing()
var("k p l j")
summation(Ynm(k, m+p, theta, phi), (k, abs(l-j), l+j))
---------------------------------------------------------------------------
ValueError                                Traceback (most recent call last)
/home/cpchung/anaconda/lib/python2.7/site-packages/IPython/core/formatters.pyc in __call__(self, obj)
    328                 pass
    329             else:
--> 330                 return printer(obj)
    331             # Finally look for special method names
    332             method = _safe_get_formatter_method(obj, self.print_method)

/home/cpchung/anaconda/lib/python2.7/site-packages/sympy/interactive/printing.pyc in _print_latex_png(o)
    123                 if latex_mode != 'inline':
    124                     s = latex(o, mode='inline')
--> 125                 return _matplotlib_wrapper(s)
    126 
    127     def _print_latex_matplotlib(o):

/home/cpchung/anaconda/lib/python2.7/site-packages/sympy/interactive/printing.pyc in _matplotlib_wrapper(o)
     86         o = o.replace(r'\operatorname', '')
     87         o = o.replace(r'\overline', r'\bar')
---> 88         return latex_to_png(o)
     89 
     90     def _can_print_latex(o):

/home/cpchung/anaconda/lib/python2.7/site-packages/IPython/lib/latextools.pyc in latex_to_png(s, encode, backend, wrap)
     85     else:
     86         raise ValueError('No such backend {0}'.format(backend))
---> 87     bin_data = f(s, wrap)
     88     if encode and bin_data:
     89         bin_data = encodestring(bin_data)

/home/cpchung/anaconda/lib/python2.7/site-packages/IPython/lib/latextools.pyc in latex_to_png_mpl(s, wrap)
    104     mt = mathtext.MathTextParser('bitmap')
    105     f = BytesIO()
--> 106     mt.to_png(f, s, fontsize=12)
    107     return f.getvalue()
    108 

/home/cpchung/anaconda/lib/python2.7/site-packages/matplotlib/mathtext.pyc in to_png(self, filename, texstr, color, dpi, fontsize)
   3092         image in pixels.
   3093         """
-> 3094         rgba, depth = self.to_rgba(texstr, color=color, dpi=dpi, fontsize=fontsize)
   3095         numrows, numcols, tmp = rgba.shape
   3096         _png.write_png(rgba.tostring(), numcols, numrows, filename)

/home/cpchung/anaconda/lib/python2.7/site-packages/matplotlib/mathtext.pyc in to_rgba(self, texstr, color, dpi, fontsize)
   3057             image in pixels.
   3058         """
-> 3059         x, depth = self.to_mask(texstr, dpi=dpi, fontsize=fontsize)
   3060 
   3061         r, g, b = mcolors.colorConverter.to_rgb(color)

/home/cpchung/anaconda/lib/python2.7/site-packages/matplotlib/mathtext.pyc in to_mask(self, texstr, dpi, fontsize)
   3030         assert(self._output=="bitmap")
   3031         prop = FontProperties(size=fontsize)
-> 3032         ftimage, depth = self.parse(texstr, dpi=dpi, prop=prop)
   3033 
   3034         x = ftimage.as_array()

/home/cpchung/anaconda/lib/python2.7/site-packages/matplotlib/mathtext.pyc in parse(self, s, dpi, prop)
   3003             self.__class__._parser = Parser()
   3004 
-> 3005         box = self._parser.parse(s, font_output, fontsize, dpi)
   3006         font_output.set_canvas_size(box.width, box.height, box.depth)
   3007         result = font_output.get_results(box)

/home/cpchung/anaconda/lib/python2.7/site-packages/matplotlib/mathtext.pyc in parse(self, s, fonts_object, fontsize, dpi)
   2337                         err.line,
   2338                         " " * (err.column - 1) + "^",
-> 2339                         six.text_type(err)]))
   2340         self._state_stack = None
   2341         self._em_width_cache = {}

ValueError: 
\sum_{k=\left\lvert{j - l}\right\rvert}^{j + l} Y_{k}^{m + p}\left(\theta,\phi\right)
             ^
Expected a delimiter (at char 13), (line:1, col:14)
```

**Comment by moorepants** [comment]
You can workaround this by using `init_printing(use_latex=True)`. This bug is due to matplotlib's limited LaTeX functionality and this is likely a duplicate of #9629.

**Comment by moorepants** [comment]
Fixed in #9959.
